app/models/invenio_rdm_converter.rb

```diff
@@ -0,0 +1,48 @@
+include Sufia::Export
+
+# Convert a GenericFile including metadata, permissions and version metadata into a PORO
+# so that the metadata can be exported in json format using to_json
+#
+class InvenioRdmConverter < Sufia::Export::Converter
+  # Create an instance of a GenericFile converter containing all the metadata for json export
+  #
+  # @param [GenericFile] generc_file file to be converted for export
+  def initialize(generc_file)
+    @id = generc_file.id
+    @provenance = generc_file.depositor
```

Comment:
This is obviously not correct, but I think it's a good example of how you could set a custom field for the export. I'm leaving the commented out fields so we can see what the default export was.

Reply:
Thanks for pointing this out.